src/main/kotlin/network/rs485/logisticspipes/config/ClientConfiguration.kt

```diff
@@ -45,6 +45,8 @@ class ClientConfiguration : LPSerializable {
     var renderPipeDistance = 48
     var renderPipeContentDistance = 24
 
+    var hasCraftedLPItem = false // Server Side Config option
```

Comment:
I dont understand that comment :D

Reply:
This means that even though it is called ClientConfiguration, it should probably be called PlayerConfiguration because it now contains information that is not meant for the client but the server but still specific to the player.

Reply:
I would disagree and I'd propose a subclass of `ClientConfiguration` (because that is sent to and from the client in a packet) called `PlayerConfiguration` and let that have additional info, which only the server has. Imo that would be much cleaner and no unnecessary info is sent between server and client.

Reply:
While even at the moment there is no unnecessary information send from the server to the client, I agree that making a PlayerConfiguration which subclasses ClientConfiguration sounds like a good idea. I will work on that.